WordPress/src/main/java/org/xmlrpc/android/XMLRPCClient.java

```diff
@@ -534,6 +534,12 @@ private Object callXMLRPC(String method, Object[] params, File tempFile)
                 // Detect login issues and broadcast a message if the error is known
                 switch (e.getFaultCode()) {
                     case 403:
+                        // Ignore 403 error from certain methods known for replying with incorrect error code on
+                        // lacking permissions
+                        if ("wp.getPostFormats".equals(method) || "wp.getCommentStatusList".equals(method)
+                            || "wp.getPostStatusList".equals(method) || "wp.getPageStatusList".equals(method)) {
+                            break;
```

Comment:
What happens when the break statement is reached here? Does the API call simply fail silently, and the user never sees the desired info (post formats, etc.)?